setup.py

```diff
@@ -14,21 +13,23 @@
 
 setup(
     name='pyzfscore',
-    version='0.0.1',
+    version='0.0.2',
     description='ZFS CFFI wrapper.',
     license='GPL3',
-    maintainer='Trevor Joynson',
-    maintainer_email='github@skywww.net',
-    url='http://github.com/akatrevorjay/pyzfscore',
-    packages=['pyzfscore',
-              'pyzfscore.libzfs',
-              'pyzfscore.flufl',
-              'pyzfscore.flufl.enum',
-              ],
-    #use_2to3=True,
-
+    author='Trevor Joynson',
+    author_email='github@skywww.net',
+    maintainer='Ben Cole',
```

Comment:
I don't believe this should be part of this PR. In fact, now I can't upload this package to PyPi =/